### PR TITLE
{lib}[GCCcore/11.3.0] nghttp3 v0.6.0

### PR DESCRIPTION
**Pull request by iotaka**
(created using `eb --new-pr`)

requires:
* #16972

### CODE REVIEW

**Comment by smoors** [comment]
@iotaka you already created a PR for `CUnit-2.1-3-GCCcore-11.3.0.eb`, so it's not useful to include it here as well.
please remove it again or close the CUnit PR?

File: easybuild/easyconfigs/c/CUnit/CUnit-2.1-3-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'CUnit'
+version = '2.1-3'
```
Comment (boegel) [inline easybuild/easyconfigs/c/CUnit/CUnit-2.1-3-GCCcore-11.3.0.eb:4]: @iotaka version here conflicts with what's used in https://github.com/easybuilders/easybuild-easyconfigs/pull/16972 ?

File: easybuild/easyconfigs/n/nghttp3/nghttp3-0.6.0-GCCcore-11.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+checksums = [
+    '3cd5fa107541a2dd11b87a84f251a98fae4c7b67df9d78be7343e4adce540b7f',  # v0.6.0.tar.gz
+]
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+    ('pkg-config', '0.29.2'),
+    ('CUnit', '2.1-3'),
```
Comment (boegel) [inline easybuild/easyconfigs/n/nghttp3/nghttp3-0.6.0-GCCcore-11.3.0.eb:26]: Should be `2.1.3` (resolved via https://github.com/easybuilders/easybuild-easyconfigs/pull/16972)?

**Review by boegel — DISMISSED** [review]

File: easybuild/easyconfigs/n/nghttp3/nghttp3-0.6.0-GCCcore-11.3.0.eb (hunk 2 above)
Comment (iotaka) [reply easybuild/easyconfigs/n/nghttp3/nghttp3-0.6.0-GCCcore-11.3.0.eb:26]: nope @smoors did a mistake renaming the title #16972 https://sourceforge.net/projects/cunit/files/CUnit/2.1-3/

Comment (smoors) [reply easybuild/easyconfigs/n/nghttp3/nghttp3-0.6.0-GCCcore-11.3.0.eb:26]: fixed

File: easybuild/easyconfigs/c/CUnit/CUnit-2.1-3-GCCcore-11.3.0.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/c/CUnit/CUnit-2.1-3-GCCcore-11.3.0.eb:4]: typo in the title, fixed

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Comment by smoors** [comment]
Going in, thanks @iotaka!

**Pull request merged**